docs/quickstart-docker-compose.md

```diff
@@ -0,0 +1,721 @@
+        environment:
+          MYSQL_DATABASE: myapp
+          MYSQL_PASSWORD: change_this_user_password
+          MYSQL_ROOT_PASSWORD: change_this_root_password
+          MYSQL_USER: appuser
+        image: percona/percona-server:8.0
+        networks:
+          default: null
+        ports:
+          - mode: ingress
+            target: 3306
+            published: "3306"
+            protocol: tcp
+        volumes:
+          - type: volume
+            source: mysql_data
+            target: /var/lib/mysql
+            volume: {}
+    networks:
+      default:
+        name: percona-mysql_default
+    volumes:
+      mysql_data:
+        name: percona-mysql_mysql_data
+        driver: local
+    ```
+    
+## Start the container
+
+This command starts the containers defined in your `docker-compose.yml` file in detached mode.
+
+* Executing the `docker-compose up -d` command.
+
+Running this command instructs Docker Compose to create and start the services defined in your `docker-compose.yml` file. The `-d` flag signifies "detached" mode, which means the containers will run in the background, freeing up your terminal. 
+
+Docker Compose will first pull any necessary images if they are not already present locally, create the containers, configure networks and volumes, and start the services. You can then interact with your application while the containers run in the background.
+
+**Try this now:** Ensure you are in the same directory as your `docker-compose.yml` file in your terminal and run the following command:
+
+
+```{.bash data-prompt="$"}
+docker-compose up -d
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    [+] Running 11/11
+     ✔ mysql Pulled                                                               32.3s 
+       ✔ 56631da24b0d Pull complete                                                7.5s 
+       ✔ 5aee836c3728 Pull complete                                                7.6s 
+       ✔ a5fd539367b0 Pull complete                                                7.6s 
+       ✔ fc4a4cc146b3 Pull complete                                                8.6s 
+       ✔ 7a3939b8d92c Pull complete                                               30.5s 
+       ✔ 6fdbd2a9e883 Pull complete                                               30.5s 
+       ✔ 70ac4d191dd1 Pull complete                                               30.5s 
+       ✔ 5872370b843d Pull complete                                               30.5s 
+       ✔ 8310fa1d2765 Pull complete                                               30.5s 
+       ✔ 4437564bc659 Pull complete                                               30.6s 
+    [+] Running 3/3
+     ✔ Network percona-mysql_default      Creat...                                 0.0s 
+     ✔ Volume "percona-mysql_mysql_data"  C...                                     0.0s 
+     ✔ Container percona-mysql            Started                                  0.3s 
+     ```
```

Comment:
When checking the Render preview, the code block is broken. Remove one space before closing ```